Updated .Net Core version

gurmeetsinghdke opened the pull request:
Replaced text ".NET Core 2.0" with ".NET Core 1.1 or any higher version" so that statement is valid for other higher available versions as well.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

gurmeetsinghdke committed:
Updated .Net Core version

Replaced text ".NET Core 2.0" with ".NET Core 1.1 or any higher version" so that statement is valid for other higher available versions as well.

Thraka commented on docs/core/tutorials/vb-with-visual-studio.md:
```diff
@@ -14,7 +14,7 @@ This topic provides a step-by-step introduction to building, debugging, and publ
 ## Prerequisites
 
-[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.0.
+[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 1.1 or any higher version.
```
,NET Core 1.0 and 1.1 are no longer supported, even though they can be used. And actually, 2.0 isn't supported either. I would change this to 2.1 which still has 1 or 2 years of support left.

```suggestion
[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or any higher version.
```

mairaw replied:
I'd build on top of @Thraka's suggestion and say "or later versions", instead of any higher version. That's our standard boilerplate.

Thraka replied:
Ahh yes good point @mairaw I've updated my suggestion.

gurmeetsinghdke replied:
Thanks @Thraka and @mairaw.. hope this helped.

gurmeetsinghdke committed:
Updated .Net Core version

Changed content from "with either .NET Core 1.1 or .NET Core 2.0." to ".NET Core 2.1 or later versions."
Earlier this changed was done in a separate PR. Combing both changes in same PR for easy review. 
Thanks @Thraka for suggestion.

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14365) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14365) before we can accept your contribution.<br/><br/>:x: gurmeetsinghdke [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14365)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=14365) it.</sub>

gurmeetsinghdke committed:
Update docs/core/tutorials/vb-with-visual-studio.md

Changes suggested by @Thraka

Co-Authored-By: Andy De George <2672110+Thraka@users.noreply.github.com>

gurmeetsinghdke commented on docs/core/tutorials/vb-with-visual-studio.md:
```diff
@@ -14,7 +14,7 @@ This topic provides a step-by-step introduction to building, debugging, and publ
 ## Prerequisites
 
-[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.0.
+[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 1.1 or any higher version.
```
Thanks @Thraka. Commited changes suggested by you and also signed CLA.

Thraka commented:
@gurmeetsinghdke Thanks a bunch for improving the docs!
